src/Logging.jl

```diff
@@ -21,23 +23,66 @@ catch
     Base.isless(x::LogLevel, y::LogLevel) = isless(x.val, y.val)
 end
 
+function Base.convert(::Type{LogLevel}, x::String)
+    for lvl in LogLevel
+        if lvl[1] == symbol(x)
+            return LogLevel(lvl[2])
+        end
+    end
+    error(InexactError())
+end
+
+@enum LogFacility LOG_KERN LOG_USER LOG_MAIL LOG_DAEMON LOG_AUTH LOG_SYSLOG LOG_LPR LOG_NEWS LOG_UUCP LOG_CRON LOG_AUTHPRIV LOG_LOCAL0=16 LOG_LOCAL1 LOG_LOCAL2 LOG_LOCAL3 LOG_LOCAL4 LOG_LOCAL5 LOG_LOCAL6 LOG_LOCAL7
+
+try
+    LOG_KERN < LOG_USER
+catch
+    Base.isless(x::LogFacility, y::LogFacility) = isless(x.val, y.val)
+end
+
+type SysLog
+    socket::UdpSocket
```

Comment:
Would there be an expectation for logging to a local syslog server via /dev/log or a local socket? It might be worth explicitly highlighting that this adds UDP syslog support only.